galaxy.yml

```diff
@@ -0,0 +1,51 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: ara
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: ara
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
```

Comment:
For the sake of simplicity I would sync this version number with the one from ara -- the current latest release is 1.1.0